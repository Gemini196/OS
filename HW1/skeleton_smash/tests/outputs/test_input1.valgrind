==41223== Memcheck, a memory error detector
==41223== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==41223== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==41223== Command: /home/student/Desktop/OS/HW1/skeleton_smash/smash
==41223== Parent PID: 41219
==41223== 
==41223== 
==41223== FILE DESCRIPTORS: 6 open at exit.
==41223== Open file descriptor 1: /home/student/Desktop/OS/HW1/skeleton_smash/tests/outputs/test_input1.out
==41223==    at 0x54EF957: dup2 (syscall-template.S:78)
==41223==    by 0x1109FD: main (smash.cpp:41)
==41223== 
==41223== Open file descriptor 0:
==41223==    at 0x54EF957: dup2 (syscall-template.S:78)
==41223==    by 0x1109AB: main (smash.cpp:33)
==41223== 
==41223== Open file descriptor 5:
==41223==    at 0x54EF927: dup (syscall-template.S:78)
==41223==    by 0x110B4B: __static_initialization_and_destruction_0(int, int) (smash.cpp:9)
==41223==    by 0x110B67: _GLOBAL__sub_I_stdout_dp (smash.cpp:57)
==41223==    by 0x110BEC: __libc_csu_init (in /home/student/Desktop/OS/HW1/skeleton_smash/smash)
==41223==    by 0x5400C17: (below main) (libc-start.c:266)
==41223== 
==41223== Open file descriptor 4: /home/student/Desktop/OS/HW1/skeleton_smash/tests/outputs/test_input1.out
==41223==    at 0x54EF927: dup (syscall-template.S:78)
==41223==    by 0x110B3B: __static_initialization_and_destruction_0(int, int) (smash.cpp:8)
==41223==    by 0x110B67: _GLOBAL__sub_I_stdout_dp (smash.cpp:57)
==41223==    by 0x110BEC: __libc_csu_init (in /home/student/Desktop/OS/HW1/skeleton_smash/smash)
==41223==    by 0x5400C17: (below main) (libc-start.c:266)
==41223== 
==41223== Open file descriptor 3: /home/student/Desktop/OS/HW1/skeleton_smash/tests/outputs/test_input1.valgrind
==41223==    <inherited from parent>
==41223== 
==41223== Open file descriptor 2: /home/student/Desktop/OS/HW1/skeleton_smash/tests/outputs/test_input1.err
==41223==    <inherited from parent>
==41223== 
==41223== 
==41223== HEAP SUMMARY:
==41223==     in use at exit: 0 bytes in 0 blocks
==41223==   total heap usage: 36 allocs, 36 frees, 83,826 bytes allocated
==41223== 
==41223== All heap blocks were freed -- no leaks are possible
==41223== 
==41223== For counts of detected and suppressed errors, rerun with: -v
==41223== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
