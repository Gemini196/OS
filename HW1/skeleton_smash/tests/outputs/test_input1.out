smash> smash> hello> smash: got ctrl-C
smash: process 47139 was killed
smash: got ctrl-C
hello> smash: got ctrl-C
smash: process 0 was killed
smash: got ctrl-C
smash: got ctrl-C
smash: got ctrl-Z
